src/test/kotlin/org/rust/lang/core/completion/RsCompletionTest.kt

```diff
@@ -1199,4 +1199,42 @@ class RsCompletionTest : RsCompletionTestBase() {
     """, """
         fn foo(f: &FnOnce(/*caret*/)) {}
     """)
+
+    fun `test complete true in variable decl`() = doSingleCompletion("""
+        fn main() { let x: bool = tr/*caret*/; }
+    """, """
+        fn main() { let x: bool = true/*caret*/; }
+    """)
+
+    fun `test complete false in variable decl`() = doSingleCompletion("""
+        fn main() { let x: bool = fal/*caret*/; }
+    """, """
+        fn main() { let x: bool = false/*caret*/; }
+    """)
+
+    fun `test no true completion in u8 variable decl`() = checkNoCompletion("""
+        fn main() { let x: u8 = tr/*caret*/; }
+    """)
+
+    fun `test no false completion in u8 variable decl`() = checkNoCompletion("""
+        fn main() { let x: u8 = fal/*caret*/; }
+    """)
+
+    fun `test complete true in condition`() = doSingleCompletion("""
+        fn main() { if tr/*caret*/ }
+    """, """
+        fn main() { if true/*caret*/ }
+    """)
+
+    fun `test complete false when assigning to bool variable`() = doSingleCompletion("""
```

Comment:
The test name uses false, but the text completes true :)

Reply:
Fixed!